notebooks/03_global_forecasting.ipynb

```diff
@@ -20,9 +20,7 @@
     "* Foundation Models\n",
```

Comment:
<p>I would assume that the license changes is not a reason for changes, since it is only affecting the model weights and not the modeling code.</p><p> </p><p>Thus, I would remove it. </p><p> </p><p>I think a reason for MOIRAI are also dependency issues</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-euroscipy2024/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>